server/routes/prediction.routes.js

```diff
@@ -360,19 +360,17 @@ function normalizeMatchFilter(filter, field){
  */
 /** @namespace filter.numDocs */
 async function getPredictions (filter, user) {
-
   try {
+    logger.debug("Starting getPredictions", { filter, userAgency: user?.agency, userRole: user?.userRole })
+    
     let first = filter.first || 0
     let max_fetch_rows = filter.rows || configuration.getConfig("defaultMaxPredictions", 1000)
 
-
     if ( user === undefined || user.agency === undefined || user.userRole === undefined ) {
+      logger.warn("Missing user information - returning empty result")
```

Comment:
Good logging message. If you see more areas to adding logging in future updates then go for it.